andela/travel_tool_front
# #162152454 Delete or edit maintenance schedule

#### What does this PR do?
This PR ensures that a maintenance schedule can be deleted or edited.

#### Description of Task to be completed?
- A travel admin can edit maintenance records.
- A travel admin can delete a maintenance record.

#### How should this be manually tested?
- Using your terminal, clone this repo -``` git clone https://github.com/andela/travel_tool_front.git``` 
- Checkout to `ft-edit-delete-maintenance-record-162152454`
- Using your terminal, clone this repo -``` git clone https://github.com/andela/travel_tool_back.git``` 
- Checkout to `ft-restore-or-delete-room-maintenance-162152454`
- Assign yourself the travel admin role by changing your roleId to 29187 in the users table
- Start the application using 'make start'
- Navigate to the `Residence` tab then click `Manage`
- Click on one of the guesthouses.
- On the timeline vertical axis click the horizontal ellipsis and click on `Unavailable`.
- Fill in the maintenance schedule details on the modal.
- Click the maintenance schedule on the timeline and you will be presented with two options to delete or edit the maintenance schedule.
- To edit click the `edit` button and fill in the modal to edit the maintenance schedule.
- To delete the schedule click on the delete button and confirm on the modal.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162152454](https://www.pivotaltracker.com/story/show/162152454)

#### Screenshots (if appropriate)
<img width="1439" alt="screenshot 2018-11-28 at 16 17 22" src="https://user-images.githubusercontent.com/12511966/49154348-33056380-f329-11e8-906c-759013287260.png">
<img width="1440" alt="screenshot 2018-11-28 at 16 17 33" src="https://user-images.githubusercontent.com/12511966/49154350-339dfa00-f329-11e8-87f4-1363ff77da62.png">
<img width="1439" alt="screenshot 2018-11-28 at 16 17 44" src="https://user-images.githubusercontent.com/12511966/49154351-339dfa00-f329-11e8-8e65-89cef769e16f.png">

#### Questions:
